plugins/chat/assets/javascripts/discourse/components/chat/thread/participants.hbs

```diff
@@ -11,11 +11,11 @@
           @showPresence={{false}}
         />
       {{/each}}
-      {{#if @thread.preview.otherParticipantCount}}
-        <div class="chat-thread-participants__other-count">
-          +{{@thread.preview.otherParticipantCount}}
-        </div>
-      {{/if}}
     </div>
+    {{#if @thread.preview.otherParticipantCount}}
+      <div class="chat-thread-participants__other-count">
+        +{{@thread.preview.otherParticipantCount}}
```

Comment:
This is a good candidate for a i18n string, which would effectively be `+{{count}}`, since on rtl languages it probably should be reversed 😃

Reply:
Fixed this up :)